aws_account_policies/aws_resource_required_tags.yml

```diff
@@ -392,6 +392,7 @@ Tests:
          "PasswordLastUsed": null,
          "Path": "/",
          "PermissionsBoundary": null,
+         "ResourceType": "AWS.IAM.User",
```

Comment:
whoops; forgot I snuck this in this PR.  This was another resource type I missed in a previous fix.